python_modules/dagster/dagster/_core/utils.py

```diff
@@ -130,3 +130,11 @@ class InheritContextThreadPoolExecutor(ThreadPoolExecutor):
     def submit(self, fn, *args, **kwargs):
         ctx = copy_context()
         return super().submit(ctx.run, fn, *args, **kwargs)
+
+    @property
+    def max_workers(self) -> int:
+        return super()._max_workers
+
+    @property
+    def currently_running_requests(self) -> int:
+        return super()._work_queue.qsize()
```

Comment:
We type all our threadpools with `: ThreadPoolExecutor` instead of this more specific `InheritContextThreadPoolExecutor` so these properties won't show. 

Maybe just keep these as private helper methods in internal instead?